src/main/kotlin/commands/IssueCommand.kt

```diff
@@ -0,0 +1,47 @@
+@file:Suppress("BlockingMethodInNonBlockingContext") //shut
+
+package commands
+
+import AuthConfig
+import Command
+import classes.issue.Issue
+import arg
+import com.google.gson.Gson
+import doesLater
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import string
+
+/**
+ * @author sourTaste000
+ * @since 2020/9/8
+ */
+object IssueCommand : Command("issue") {
+    init {
+        string("user") {
+            string("repoName") {
+                string("issueNum") {
+                    doesLater { context ->
+                        val user: String = context arg "user"
+                        val repoName: String = context arg "repoName"
+                        val issueNum: String = context arg "issueNum"
+                        val githubToken = FileManager.readConfigSafe<AuthConfig>(ConfigType.AUTH, true)?.githubToken
```

Comment:
this should return and send an error in the channel if the token is null (see rules command for a concise oneliner example)